# Configuration for bicycle planning
kinematicModel: Bicycle

#Reached parameters
deltaX: 0.3
deltaTheta: 1.57

motion_primitives:
  minU: [0, -0.78539816339]
  maxU: [1, 0.78539816339]
  discretization: 20
